CHANGES/831.removal

```diff
@@ -0,0 +1,2 @@
+Removed "--sqlite-metadata/--no-sqlite-metadata" option from `pulp rpm repository update` command.
+It was phased out from the pulp_rpm plugin.
```

Comment:
Can you update the changelog according to the change please?

Reply:
How is that?
> Added a version-dependency guard of `pulp_rpm<=3.4.0` to the `--sqlite-metadata/--no-sqlite-metadata` option from `pulp rpm repository update`  command, as it was phased out from the plugin.

And this is still considered a ".removal", right?

Reply:
Let's keep it a ".removal", as it's a breaking change for the user.
I would try to make it sound less technical:

> Marked option `--sqlite-metadata` on `pulp rpm repository update` unavailable for `pulp_rpm>=3.4`, as it was removed there.

Another (could be very technical again; it's a different audience) changelog should go into `CHANGES/pulp-glue` then.